### PR TITLE
SVA-281 | Hide welcome screen after first time the user sees it.

### PR DESCRIPTION
**Pull request by kelsiesmurphy**
- [Description](#description)
  - [Type of change](#type-of-change)
- [Testing locally](#testing-locally)
- [Checklist](#checklist)

# Description
### Problem
As a user, I only want to see the app welcome screen the first time I use the app.

### Solutions
Needs an update to the WelcomeButton handlePress function, to call the Redux dispatch method like in the ProfileContainer onChangeSplash function (and will need to access welcomeState using useSelector like in ProfileContainer).

Fixes # SVA-281

## Type of change

Please delete options that are not relevant

- [x] New Feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have removed any unnecessary comments or console logging
- [ ] I have made corresponding changes to the documentation (if required)
- [ ] I have addressed accessibility, if needed
- [x] I have followed best practices, e.g. NativeBase approaches and theming
- [x] I have checked the app in dark mode, if making front-end design changes
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by joefhall** [comment]
Nice one, thanks @kelsiesmurphy!  I've just spotted this is failing the ES Lint checks -- not because of anything you've done 🙂  but because of some formatting issues with the existing code (I think the code in that file was added before we set up the automated checks).

You'll need to fix those or GitHub won't let you merge this in -- if you click on 'Details' below next to the check that's failing, it should tell you what needs changing.

**Comment by kelsiesmurphy** [comment]
Oops, thanks for that Joe, they're now passing. Let me know if you need more detail for the JSDocs side of things.

File: app/ios/Podfile.lock [hunk 1]
```diff
@@ -269,18 +269,18 @@ PODS:
     - React-jsi (= 0.65.1)
```
Comment (joefhall) [inline app/ios/Podfile.lock:269]: As this PR isn't about making any package changes, I don't think you should need to include this file?

File: app/ios/STA.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -994,7 +994,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
-				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "arm64 ";
+				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
```
Comment (joefhall) [inline app/ios/STA.xcodeproj/project.pbxproj:997]: Also not sure these changes should be included in this PR.  If they're general changes that do need to be made for the iOS setup, I think that should go in a separate PR (and so it can be flagged for testing by other Mac users, etc).

File: app/src/Components/Welcome/WelcomeButton.tsx [hunk 3]
```diff
@@ -1,15 +1,23 @@
-//'Get Started' button allows user to go to main part of app from any slide
-
+/**
+ * @file 'Get Started' button allows user to go to main part of app from any slide
+ */
 import React from 'react'
-import {TouchableOpacity, Text} from 'react-native'
+// import { TouchableOpacity, Text } from 'react-native'
```
Comment (joefhall) [inline app/src/Components/Welcome/WelcomeButton.tsx:5]: Please can you remove this line if no longer needed, rather than commenting it out

**Review by joefhall — CHANGES_REQUESTED** [review]
Nice work, thanks @kelsiesmurphy!  I spotted a few little things which have crept in, if you could take a look -- but the functionality change itself looks good to me 👍

File: app/src/Components/Welcome/WelcomeButton.tsx (hunk 3 above)
Comment (kelsiesmurphy) [reply app/src/Components/Welcome/WelcomeButton.tsx:5]: I'm not sure what this is actually, both were flagged in the linting checks for going through unused, but I'll remove them for now.

File: app/ios/Podfile.lock (hunk 1 above)
Comment (kelsiesmurphy) [reply app/ios/Podfile.lock]: This is no longer included.

File: app/src/Components/Welcome/WelcomeButton.tsx (hunk 3 above)
Comment (kelsiesmurphy) [reply app/src/Components/Welcome/WelcomeButton.tsx]: This line has been removed

**Review by joefhall — APPROVED** [review]
Good work @kelsiesmurphy -- thanks for all of this! 🙌

**Review by PammyLo — APPROVED** [review]
works fine on iOS too :)
Great work @kelsiesmurphy, thanks
